inst/extdata/additional_summartion_checks.csv

```diff
@@ -0,0 +1,11 @@
+parent;                           child;                                     factor
```

Comment:
There is a typo in the filename, an additional `r` in `summation`. I would rather rename it to `summation_groups_REMIND.csv` similar to the filenames in `piamInterfaces`, also so we don't have to rename it once we eventually eliminate the `|+|` notation ;)